public/app/features/query-library/api/query.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseQueryFn } from '@reduxjs/toolkit/query/react';
+import { lastValueFrom } from 'rxjs';
+
+import { getBackendSrv, isFetchError } from '@grafana/runtime/src/services/backendSrv';
+
+import { DataQuerySpecResponse } from './types';
+
+export const BASE_URL = '/apis/peakq.grafana.app/v0alpha1/namespaces/default/querytemplates/';
+
+/**
```

Comment:
I might put some warnings in here about how it is alpha and an unstable API.

Reply:
Added in [bc7f1c7](https://github.com/grafana/grafana/pull/86897/commits/bc7f1c7f99b5072ed1aff73c762320a5f19fd2e5)